pysha256sum

```diff
@@ -0,0 +1,6 @@
+#! /usr/bin/env python
```

Comment:
why have this here as a separate file, but patchify itself keeps it as a string?

Reply:
It's needed for the patch creation wizard. I was pondering what to do about that, even considering pulling it out of patchify itself, but that would mean patchify (the script) was no longer standalone. Keeping it separate allows the patch creation wizard to work while still meaning that if a user just downloads patchify itself they'll be able to (once we add some) get manager-updates.